pkg/apis/serving/v1/revision_defaults.go

```diff
@@ -93,7 +93,15 @@ func (rs *RevisionSpec) SetDefaults(ctx context.Context) {
 		if rs.PodSpec.Containers[idx].ReadinessProbe.SuccessThreshold == 0 {
 			rs.PodSpec.Containers[idx].ReadinessProbe.SuccessThreshold = 1
 		}
+		if rs.PodSpec.Containers[idx].ReadinessProbe.PeriodSeconds != 0 {
+			if rs.PodSpec.Containers[idx].ReadinessProbe.FailureThreshold == 0 {
+				rs.PodSpec.Containers[idx].ReadinessProbe.FailureThreshold = 3
```

Comment:
Where are those defaults coming from?

Reply:
@vagababov  https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

> timeoutSeconds: Number of seconds after which the probe times out. Defaults to 1 second. Minimum value is 1.
> failureThreshold: When a Pod starts and the probe fails, Kubernetes will try failureThreshold times before giving up. Giving up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready. Defaults to 3. Minimum value is 1.

Reply:
Should they be named constant with a comment where you got them from?